lib/Alchemy/Phrasea/SearchEngine/Elastic/ElasticsearchSettingsFormType.php

```diff
@@ -104,15 +105,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     "100 values"     => 100,
                     "all values"     => -1
                 ];
+                /*To do : This is just a test but  the filter should depends on selected choice and not that*/
+               $attr = [  'class' => 'aggregate' ];
```

Comment:
not a good "if" to test dispatch to left or right since only 1 hardcoded facet have "choice" : flashfired).
better : random...